seedsource_core/django/seedsource/management/commands/create_vector_tiles.py

```diff
@@ -60,24 +60,26 @@ def handle(self, *args, **options):
             if process.returncode == 0:
                 self.stdout.write(self.style.SUCCESS("Success\n"))
                 outputIndex.append({
-                    'name': source,
+                    'name': source[0],
                     'type': 'vector',
-                    'urlTemplate': f'http://localhost:3333/services/seedzones/{source}' + "/tiles/{z}/{x}/{y}.png",
+                    'urlTemplate': f'services/seedzones/{formatted_source}' + "/tiles/{z}/{x}/{y}.png",
                     'zIndex': 1,
                     'displayed': False
-                })
+                    })
             else:
-                errors.append(source)
+                errors.append(formatted_source)
                 self.stdout.write(self.style.ERROR("Error\n"))
 
-        self._write_out("Creating index..")
-        with open(os.path.join(tiles_dir, "index.js"), "w") as f:
-            f.write("[\n")
+        self._write_out("Creating tilesIndex..")
+        with open(os.path.join(tiles_dir, "tilesIndex.js"), "wb") as f:
```

Comment:
This is text, not binary, so you should open the file as `'w'` instead of `'wb'`, which will also save you from having to do `.encode()` everywhere.

```python
with open(os.path.join(tiles_dir, 'tilesIndex.js'), 'w') as f:
    f.write('export default [\n')
```

Reply:
Also, does this need to be a JS file. How about just exporting `outputIndex` as JSON:

```python
with open(os.path.join(tiles_dir, 'tilesIndex.js'), 'w') as f:
    f.write(json.dumps(outputIndex))
```

https://webpack.js.org/loaders/json-loader/